### PR TITLE
A slight grammatical change to avoid confusion

### PR DESCRIPTION
**Pull request by vinod827**
A slight grammatical change to avoid confusion while reading about the Network Policy in a specific namespace.

### CODE REVIEW

File: content/en/docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Network policies are implemented by the [network plugin](/docs/concepts/extend-k
 
 By default, pods are non-isolated; they accept traffic from any source.
 
-Pods become isolated by having a NetworkPolicy that selects them. Once there is any NetworkPolicy in a namespace selecting a particular pod, that pod will reject any connections that are not allowed by any NetworkPolicy. (Other pods in the namespace that are not selected by any NetworkPolicy will continue to accept all traffic.)
+Pods become isolated by having a NetworkPolicy that selects them. Once there is any NetworkPolicy in a namespace selecting a particular pod, that pod will reject any connections that are not allowed by that NetworkPolicy. (Other pods in the namespace that are not selected by any NetworkPolicy will continue to accept all traffic.)
```
Comment (caseydavenport) [inline content/en/docs/concepts/services-networking/network-policies.md:34]: if anything it should probably be "those network policies".

"that NetworkPolicy" is singular, and what this is trying to say is that any connections which are not allowed by at least one NetworkPolicy will be rejected.


Probably wrong to say "that pod will reject" as well.

How about this:

> Pods become isolated as soon as a NetworkPolicy's spec.PodSelector selects them. Once there is any NetworkPolicy selecting a given pod, connections to and from that pod will fail unless they are allowed by at least one NetworkPolicy.

**Comment by bradtopol** [comment]
Hi @vinod827 thank you for your contribution. Please sign the CLA so we can accept your PR for review.
/assign

File: content/en/docs/concepts/services-networking/network-policies.md (hunk 1 above)
Comment (Aut0R3V) [reply content/en/docs/concepts/services-networking/network-policies.md:34]: I think @caseydavenport  suggestion makes sense.

Comment (caseydavenport) [reply content/en/docs/concepts/services-networking/network-policies.md:34]: It probably needs a little more clarification, now that I read it a second time.

The main thing being that ingress/egress are independent of each other.

> Pods become isolated as soon as a NetworkPolicy's spec.PodSelector selects them. Once there is any Ingress NetworkPolicy selecting a given pod, connections to that pod will fail unless they are allowed by at least one ingress rule. Similarly, once any Egress NetworkPolicy selects a given pod, connections from that pod will fail unless they are allowed by at least one egress rule.

**Review by Arhell — COMMENTED** [review]
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by vinod827** [comment]
thanks, I have signed the CNCF ICLA

**Comment by vinod827** [comment]
> Hi @vinod827 thank you for your contribution. Please sign the CLA so we can accept your PR for review.
> 
> /assign

Hi, I have signed the CLA. Seems there is an issue with the required label as one of the check has not passed.

**Comment by bradtopol** [comment]
Hi @caseydavenport please serve as the technical reviewer for this and add an /lgtm when you feel it is correct and I'll take care of the rest.

Thanks

**Comment by vinod827** [comment]
> Hi @caseydavenport please serve as the technical reviewer for this and add an /lgtm when you feel it is correct and I'll take care of the rest.
> 
> Thanks

Hi @caseydavenport / @danwinship, I would really appreciate it if you can please add the label for the PR build to pass. thank you

**Comment by danwinship** [comment]
The proposed change is incorrect. See Casey's comments.

**Review by vinod827 — COMMENTED** [review]
Incorporated the review comments given in the PR.

**Comment by vinod827** [comment]
> The proposed change is incorrect. See Casey's comments.

@danwinship , thanks, I have made changes as per the review comments given @caseydavenport 

/assign @bradtopol

**Comment by vinod827** [comment]
please add the necessarily approved label, thanks 
https://prow.k8s.io/pr?query=is%3Apr+repo%3Akubernetes%2Fwebsite+author%3Avinod827+head%3Amaster

**Comment by vinod827** [comment]
@bradtopol / @caseydavenport , please add the label to approve this PR

**Pull request closed without merging**